packages/core/src/services/shellExecutionService.ts

```diff
@@ -89,13 +89,16 @@ export class ShellExecutionService {
     abortSignal: AbortSignal,
   ): ShellExecutionHandle {
     const isWindows = os.platform() === 'win32';
-    const shell = isWindows ? 'cmd.exe' : 'bash';
-    const shellArgs = [isWindows ? '/c' : '-c', commandToExecute];
 
-    const child = spawn(shell, shellArgs, {
+    const child = spawn(commandToExecute, [], {
```

Comment:
This definitely feels risky given the precedent of touching the spawn function. Give it a shot but please keep an eye cross-plat and react as necessary